app/helpers/url_helper.rb

```diff
@@ -0,0 +1,5 @@
+module UrlHelper
+  def url_with_default_protocol(url)
+    url.match(/^http/i) ? url : "https://#{url}"
```

Comment:
@chrisjwelly probably can use this helper later as currently the QuillJS link will by default open "nuansacp.com/{url}" by default if the "http" is not specified at the beginning of the line